samples/algorithms/sudoku-grover/README.md

```diff
@@ -0,0 +1,200 @@
+---
+page_type: sample
+languages:
+- qsharp
+- csharp
+products:
+- qdk
+description: "This sample uses Grover's search algorithm to solve Sudoku puzzles."
+
+---
+
+# Solving Sudoku using Grover's Algorithm
+
+
+     This program demonstrates solving Sudoku puzzle using Grovers algorithm.
+     
+     The code supports both 4x4 and 9x9 Sudoku puzzles.
+          
```

Comment:
Due to this being indented by four spaces, GitHub formatting is picking it up as monospace-formatted. It would be easier to read formatted as prose, such that I'd suggest de-indenting prose lines such as this one.

Reply:
i did indent it on purpose so that the graphs/diagrams were displayed correctly.